### PR TITLE
Add `hann` export to be equal to `hanning`

### PR DESCRIPTION
**Pull request by staticfloat**
Since the original author's name was "Julius Von Hann", and referring to it as "hanning" is done only to draw similarities to the similar "hamming" window, let's export it under both common names.

### CODE REVIEW

File: src/windows.jl [hunk 1]
```diff
@@ -159,7 +160,7 @@ The window is defined by sampling the continuous function:
 
 in the range `[-0.5, 0.5]`
 
-The `hanning` window satisfies the Constant Overlap-Add (COLA) property with an
+The `hanning` window satisfies the Constant Overlap-Add (COLA) property with a
```
Comment (rob-luke) [inline src/windows.jl:163]: Perhaps explain in the documentation why it is exported as hann too (similar to the explanation in the PR). 1) it's interesting piece of knowledge, and 2) so users can find it.

File: src/windows.jl [hunk 2]
```diff
@@ -176,6 +177,7 @@ function hanning(n::Integer; padding::Integer=0, zerophase::Bool=false)
         0.5*(1+cos(2pi*x))
     end
 end
+hann = hanning
```
Comment (giordano) [inline src/windows.jl:180]: Shouldn't it be a const?
```suggestion
const hann = hanning
```

**Comment by rob-luke** [comment]
Sorry for string reviewing. But could you please add a small test?

File: src/windows.jl [hunk 3]
```diff
@@ -166,6 +166,11 @@ hop of 0.5, which means that adding together a sequence of delayed windows with
 a signal from a number of overlapping frames (each with a roughly rectangular
 window), to eliminate windowing amplitude modulation.
 
+While the this windowing function was originally named after its inventor
```
Comment (martinholters) [inline src/windows.jl:169]:
```suggestion
While this windowing function was originally named after its inventor
```

**Review by rob-luke — APPROVED** [review]

**Comment by ViralBShah** [comment]
Good to merge?

**Pull request merged**